utils/metrics/src/lib.rs

```diff
@@ -40,6 +40,7 @@ async fn collect_metrics(req: tide::Request<Registry>) -> tide::Result {
         .encode(&metric_families, &mut metrics)
         .expect("Encoding Prometheus metrics must succeed.");
     Ok(tide::Response::builder(tide::StatusCode::Ok)
+        .header("content-type", "text/plain; version=0.0.4; charset=utf-8")
```

Comment:
Hey, thanks for the submission. Does the Prometheus server you are using to scrape forest require these headers?

Reply:
Actually, not sure it needs or not. But when you open the http://127.0.0.1:6116/metrics in the browser, you will get the download page instead of metrics because of the  `application/octet-stream` type.

Also, the specs say the following: 
> A missing version value will lead to a fall-back to the most recent text format version

Reply:
I see.

Can you update to use the appropriate constants exposed by the stdlib?
https://docs.rs/http/0.1.5/http/header/constant.CONTENT_TYPE.html